### PR TITLE
Tiny store crate cleanup

### PR DESCRIPTION
**Pull request by hashmap**

### CODE REVIEW

File: store/src/types.rs [hunk 1]
```diff
@@ -28,6 +28,16 @@ use libc::{ftruncate64, off64_t};
 use core::core::hash::Hash;
 use core::ser;
 
+macro_rules! read_bitmap {
```
Comment (antiochp) [inline store/src/types.rs]: (personal bias against macros) Can we do this with a simple fn and avoid having to introduce a macro?

**Comment by hashmap** [comment]
Sorry, tap the wrong button:(. Bitmap is unsized, would have to Box it to return from function, kinda ugly. Also a risk to copy a huge chunk of memory, hence macros

**Comment by hashmap** [comment]
Ah, don‘t merge, forgot to use it in couple places.

**Comment by antiochp** [comment]
But we return a bitmap in a couple of place like this without having to box it up? 

https://github.com/mimblewimble/grin/blob/3b5a39dd42242263f25e031f75e652d122f6a9af/store/src/leaf_set.rs#L92

**Comment by hashmap** [comment]
@antiochp yeah, my bad, Bitmap is sized, not sure why I thought differently. I've replaced it with a function.

**Pull request merged**